.codepipeline/buildspec-tests-build.yml

```diff
@@ -43,8 +55,9 @@ phases:
       - make test-cbc-integration; exit 0
       - make test-platform-admin-flow; exit 0
       - make test-authentication-flow; exit 0
-      - make test-top-rail-services; exit 0
+      # - make test-top-rail-services; exit 0
```

Comment:
Is there a reason we commented this out rather than just removing it?

Reply:
This is a valid test that is waiting for the alerting work to suppress the output in the support channel. It's commented out here to reduce the number of unhelpful notifications.